src/components/Swap/Toolbar/Caption.tsx

```diff
@@ -145,10 +133,10 @@ export interface TradeProps {
   impact?: PriceImpact
 }
 
-export function Trade({ trade, outputUSDC, impact, gasUseEstimateUSD }: TradeProps & GasEstimateProp) {
+export function Trade({ trade, outputUSDC, impact, gasUseEstimateUSD }: TradeProps & GasEstimateProps) {
```

Comment:
`TradeProps` should just extend `GasEstimateProps` instead of using an intersection type.